src/test/java/com/snowflake/kafka/connector/internal/IngestionServiceIT.java

```diff
@@ -62,4 +64,60 @@ public void ingestFileTest() throws Exception {
 
     assert ingestService.getStageName().equals(stage);
   }
+
+  @Ignore
+  @Test
+  public void ingestFileWithClientInfoTestSuccessful() throws Exception {
+    String file = "{\"aa\":123}";
+    String fileName = FileNameUtils.fileName(TestUtils.TEST_CONNECTOR_NAME, table, 0, 0, 1);
```

Comment:
can you add comment here for filename. 
I think the parameters are table, partition_no, startOffset, endOffset